i18n/locales/vi-VN/hourofcode/en.yml

```diff
@@ -0,0 +1,5922 @@
+'vi':
```

Comment:
Why was this file added?

Reply:
Ah there's an unknown character that's causing the yaml parsing to fail (string: https://crowdin.com/translate/hour-of-code/595/en-vi#2289955)

Reply:
Fixed in 678b5c4